.github/workflows/release.yml

```diff
@@ -69,21 +69,25 @@ jobs:
             qtPath: ''

```

Comment:
Could you please also make it so that the main.yml builds the linux image the same way as thte release yml.
So that mainly the deployment is the difference between those two.
Thx,

Reply:
Probably worth looking if those two can be merged completly and only the deploy step happens if the tag exists.....

Reply:
Yeah, that's a good idea. I'll rewrite the script so it replaces the deploy step rather than running cmake itself at all.

Reply:
Working on this still.

Reply:
I set it so that the main.yml compiles with the same flags as the .AppImage version for now. I can look into merging the two tasks completely later, it's too much work for this PR.